tests/test_construction.py

```diff
@@ -81,23 +81,18 @@ def test_simple_copy():
     assert m.model_fields == m2.model_fields
 
 
-@pytest.fixture(scope='session', name='ModelTwo')
-def model_two_fixture():
-    class ModelTwo(BaseModel):
-        __foo__ = PrivateAttr({'private'})
+class ModelTwo(BaseModel):
```

Comment:
The reason I did this was that pickling requires models to be defined at module level, and the last time these tests were passing was when `ModelTwo` was defined at the module level; I assumed it was moved into a fixture to prevent failures at import time and now that it didn't generate those it could just be moved back.

But in accordance with your suggestion, I have reverted this and used `create_module` to handle the pickle tests.